calculator.js

```diff
@@ -6,14 +6,69 @@ const exampleAdditionInput = {
 }
 
 const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+  // console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+  const reg = /\(\d+[-+\/*]\d+\)/;
+  let num1 = '';
+  let num2 = '';
+  if ((reg.test(promptInput.num1) === true) && (reg.test(promptInput.num2) === true)) {
+    num1 = eval(promptInput.num1); 
+    num2 = eval(promptInput.num2);
+  } else if ((reg.test(promptInput.num1) === true) && (reg.test(promptInput.num2) === false)) {
+    num1 = eval(promptInput.num1);
+    num2 = Number(promptInput.num2);
+  } else if ((reg.test(promptInput.num2) === true) && (reg.test(promptInput.num1) === false)) {
+    num2 = eval(promptInput.num2);
+    num1 = Number(promptInput.num1);
+  } else {
+    if (isNaN(Number(promptInput.num1)) && isNaN(Number(promptInput.num2))) {
+      return console.log(`${ promptInput.num1 } and ${ promptInput.num2 } are not numbers`);
+    } else if (isNaN(Number(promptInput.num1))) {
+      return console.log(`${ promptInput.num1 } is not a number`);
+    } else if (isNaN(Number(promptInput.num2))) {
+      return console.log(`${ promptInput.num2 } is not a number`);
+    } 
+    num1 = Number(promptInput.num1);
+    num2 = Number(promptInput.num2);
+  }
 
+  const operation = promptInput.operation.toLowerCase();
+
+  if (operation === 'add' || operation === '+') {
+    console.log(`${ promptInput.num1 } + ${ promptInput.num2 } = ` + (num1+num2));
+  } else if (operation === 'subtract' || operation === '-') {
+    console.log(`${ promptInput.num1 } - ${ promptInput.num2 } = ` + (num1-num2));
+  } else if (operation === 'multiply' || operation === '*') {
+    console.log(`${ promptInput.num1 } * ${ promptInput.num2 } = ` + (num1*num2));
```

Comment:
AS you menionted, consider encapsulating this functionality into helper methods. You may also consider assigning `promptInput.num1` to a variable `num1` and `promptInput.num2` to a variable `num2` to increase readability.

```
const multiply = (num1, num2) => `${num1} * ${num2} = ${num1*num2}`
console.log(mutliply(num1, num2));
```